src/Xamarin.Android.Build.Tasks/Tasks/GenerateResourceDesignerIntermediateClass.cs

```diff
@@ -19,7 +19,7 @@ public class GenerateResourceDesignerIntermediateClass : AndroidTask
 using System;
 
 namespace %NAMESPACE% {
-	public partial class Resource : %BASECLASS% {
+	public class Resource : %BASECLASS% {
```

Comment:
I'm not sure that I like this idea; I *like* `partial` classes…

Reply:
I was just testing to see if this would break any tests...it didn't 🤣